tests/test_system.py

```diff
@@ -159,21 +188,28 @@ def test_create_valid_url_encodes_url(self):
         url = system.create_valid_url('http://ex.com/path,/sdf')
         assert url == 'http://ex.com/path%2C/sdf'
 
-    def test_create_valid_url_fixes_and_encodes_url(self):
+    @pytest.mark.parametrize('original,expected', [
```

Comment:
The docs could be clearer with this (I did the same thing based on their examples) but you actually can include the nice space between the args in the string and it will still work fine. Not a big deal but it makes it just a tad easier on the eyes.